lib/aida/bot.ex

```diff
@@ -55,6 +56,16 @@ defmodule Aida.Bot do
   end
 
   @spec chat(message :: Message.t) :: Message.t
+  def chat(%Message{ content: %SystemContent{text: text}, session: session } = message) do
+    case text do
```

Comment:
##FOO will break the bot. Let's default to the normal behavior if it's not a known system command.